force-app/volunteers/reports/VolunteerReports/VolunteersByActivity.report-meta.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <columns>
+        <field>CUST_NAME</field>
+    </columns>
+    <columns>
+        <field>VolunteerActivityAssignment__c.Status__c</field>
+    </columns>
+    <columns>
+        <field>FK_CNTC_FIRST_NAME</field>
+    </columns>
+    <columns>
+        <field>FK_NAME</field>
+    </columns>
+    <description>List of Volunteer Activity Assignments, grouped by Volunteer Activity.</description>
+    <format>Summary</format>
+    <groupingsDown>
+        <dateGranularity>Day</dateGranularity>
+        <field>VolunteerActivityAssignment__c.VolunteerActivity__c</field>
+        <sortOrder>Asc</sortOrder>
+    </groupingsDown>
+    <name>Volunteers by Activity</name>
+    <params>
+        <name>co</name>
+        <value>1</value>
+    </params>
+    <reportType>CustomEntity$VolunteerActivityAssignment__c@VolunteerActivityAssignment__c.VolunteerContact__c</reportType>
+    <scope>user</scope>
```

Comment:
I see in the report builder that this one is filtering to My Volunteer Activity Assignments instead of all. I think that means that this should be `<scope>organization</scope>` instead of user.